query-engine/dmmf/Cargo.toml

```diff
@@ -13,6 +13,9 @@ schema-builder = { path = "../schema-builder" }
 indexmap = { version = "1.7", features = ["serde-1"] }
 prisma-models = { path = "../prisma-models" }
 
+wasm-bindgen = { version = "0.2.84" }
+tsify = { version = "0.4.3", features = ["json"] }
```

Comment:
(for when this is ready to review: we have prisma-fmt-wasm in order to avoid wasm-bindgen & co as dependencies on other crates — from my experience, and the problems we had with https://github.com/prisma/connection-string/ recently, wasm-bindgen is very invasive, and we should try hard to keep it only in the one specifically-for-wasm crate)

Reply:
Imagining such a comment appearing, one thing I had tried on Friday was using `cfg` attributes to conditionally include dependencies like `wasm-bindgen` and `tsify` when the compilation target's `arch` is `wasm32`. That resulted in a bunch of compilation errors I couldn't find a quick way of.

So, I figured that I would first apply this PR with a "best effort dependencies" logic, and then pair with you or someone else for streamlining how those deps are imported and making sure their respective Cargo.toml files adhere to the engines' conventions (e.g., using workspaces, etc).

Now that I'm thinking of it, we could perhaps use those dependencies only if a `wasm` feature is defined in internal crates (which we would set from `prisma-fmt-wasm` on import). Not sure what's the easiest path ahead.